packages/ses/src/lockdown-shim.js

```diff
@@ -263,6 +264,10 @@ export const repairIntrinsics = (options = {}) => {
 
   tameDomains(domainTaming);
 
+  const SharedSymbol = tameSymbolConstructor();
+  // Must happen before `makeIntrinsicsCollector()`
+  globalThis.Symbol = SharedSymbol;
```

Comment:
I think this has the unintended effect of erasing powerful symbols from the start compartment, so they can’t be delegated selectively by an application. Should we leave the start compartment’s `Symbol` in tact?

Reply:
Good suggestion. This would also make this taming more consistent with our other tamings. Will do.

Reply:
Fixed. PTAL